How To Market Your Store and Listings on OpenBazaar blog post (#113)
OpenBazaar/official_site: _posts/blog/2019-02-15/2019-02-15-how-to-market-your-store-and-listings-on-openbazaar.markdown

---
title: "How To <span class='post-title-extra'>Market Your Store and Listings</span> on OpenBazaar"
layout: post
date: '2019-02-15 08:30:00 -0600'
social_title: 'How To Market Your Store and Listings on OpenBazaar'
social_description: 'Some of the earliest feedback we heard from ecommerce sellers who were interested in OpenBazaar was that they wanted a way to easily share their store and listings on their social networks. Here is how we did that.'
---

The shift to peer-to-peer online commerce won’t happen overnight. Until this technology becomes widespread, we need tools that help the transition go more smoothly.

Some of the earliest feedback we heard from ecommerce sellers who were interested in OpenBazaar was that they wanted a way to easily share their store and listings on their social networks. To answer this we built [OpenBazaar.com](https://openbazaar.com).

#### A regular website to look into a decentralized network

> [OpenBazaar](https://openbazaar.org/download) is an application that users have to download and run on their own computers to connect to and participate in the network. It is built this way on purpose because of the decentralized nature of the network. While the tech isn’t available for a full OpenBazaar buying & selling experience in a browser yet, that doesn’t mean there can’t be just a simple website that looks into the marketplace.

Reviewer: I'd drop the "on purpose", I don't think it's needed.

Reviewer: You might make it:

`It is built this way to make the decentralization of the network possible.`

Author: Done

> That solved the simplest part of the problem: that sellers wanted to share their links easily and potential buyers—who may or may not know about OpenBazaar yet—wanted to view them in a predictable way. 

Reviewer: ```suggestion
That solved the simplest part of the problem: sellers wanted to share their links easily and potential buyers—who may or may not know about OpenBazaar yet—wanted to view them in a predictable way. 
```

> We launched [OpenBazaar.com](https://openbazaar.com) in the middle of last year and it’s been a hit. Traffic to the site is growing **around 40% month over month**. This is clearly showing us that users are interested in decentralized applications that structurally uphold some core values, but that also are accessible wherever they may be in their technological adventures.

Reviewer: ```suggestion
We launched [OpenBazaar.com](https://openbazaar.com) in the middle of last year and it’s been a hit. Traffic to the site is growing **around 40% month over month**. This shows us that users are interested in decentralized applications that uphold some core values, but that are also accessible wherever they may be in their technological adventures.
```

Author: Done

#### Sharing store & listings from within OpenBazaar

> People who want to share stores and listings easily on the web can easily grab their links from OpenBazaar.com several ways:

Reviewer: ```suggestion
People who want to share stores and listings easily on the web can grab their links from OpenBazaar.com several ways:
```

No double easily.

1. By clicking the green “View on Web” link in the address bar within their OpenBazaar client:

![OpenBazaar View on Web Link](openbazaar_view_on_web_link.png "OpenBazaar View on Web Link")

2. By copying the store ID and pasting it on the end of https://openbazaar.com/store/ like this:

![OpenBazaar PeerID Selected](openbazaar_peerid_selected.png "OpenBazaar PeerID Selected")
 
[https://openbazaar.com/store/QmcUDmZK8PsPYWw5FRHKNZFjszm2K6e68BQSTpnJYUsML7](https://openbazaar.com/store/QmcUDmZK8PsPYWw5FRHKNZFjszm2K6e68BQSTpnJYUsML7)

<br>  

#### Sharing store & listings from OpenBazaar.com

> On OpenBazaar.com there are also additional features including these social share buttons:

Reviewer: ```suggestion
On OpenBazaar.com there are additional features including these social share buttons:
```

Also and additional mean the same thing, so you only need one.

Author: Done

![OpenBazaar Widget Builder Link and Social Share Buttons](openbazaar_store_widget_builder_link.png "OpenBazaar Widget Builder Link and Social Share Buttons")

There is also [the widget](https://openbazaar.com/widget) for sellers who have websites where they want to showcase their OpenBazaar store. Simply copy the code and add it to your site wherever you can post an embed or add HTML.

![OpenBazaar Store Widget Builder](openbazaar_store_widget_builder.png "OpenBazaar Store Widget Builder")

> We are working to build a fully decentralized marketplace and social experience to promote privacy and autonomy for users around the world. It’s clear to us, though, that the road to full decentralization is made up of more decentralized steps rather than a complete jump. 

Reviewer: ```suggestion
We are working to build a fully decentralized marketplace and social experience to promote privacy and autonomy for users around the world. It’s clear to us, though, that the road to full decentralization is made up of multiple  decentralized steps rather than a complete jump. 
```

"more" is unclear, more than what? Normally it's used when comparing to something else.

> Moving from centralized systems we are displeased with is a process that will take some time and the ability to make small choices through that process is imperative. Building decentralized apps that also interface with centralized apps that people are used to using right now is important to help them better match their desires for the future with their present reality.

Reviewer: "choices" is ambiguous here. What choices are being made? Maybe "made small advances"?
